godel_surface_detection/src/services/surface_blending_service.cpp

```diff
@@ -376,6 +382,24 @@ class SurfaceBlendingService
 					std::reverse(bnd.begin(), bnd.end());
 				}
 
+				// DEBUG
+				godel_process_path::PolygonBoundaryCollection coll;
+				BOOST_FOREACH(godel_process_path::PolygonBoundary &bnd, filtered_boundaries)
+				{
+					godel_surface_detection::ProfilimeterScanParams param;
+					param.width = 0.04;
+					param.overlap = 0.0;
+
+					// Filter and reverse boundaries
+					coll.push_back(godel_surface_detection::generateProfilimeterScanPath(bnd, param));
+				}
+				visualization_msgs::MarkerArray scan_markers;
+				std_msgs::ColorRGBA blue;
+				blue.b = 1.0;
+				blue.r = 0.0;
+				blue.g = 0.0;
```

Comment:
If the alpha channel doesn't get set I think it is automatically initialized to 0.